spec/unit/filter_option_parser_spec.rb

```diff
@@ -32,26 +32,26 @@
       it "handles simple filter definitions" do
         filter = subject.parse(
           "inspect",
-          "exclude" => "/unmanaged_files/files/name=/foo"
+          "exclude" => "/unmanaged_files/name=/foo"
         )
 
         expect(filter.to_array).to match_array([
-          "/unmanaged_files/files/name=/foo"
+          "/unmanaged_files/name=/foo"
         ])
       end
 
       it "reads filter from a filter definition file" do
         exclude_file = given_dummy_file("exclude_file")
         File.write(exclude_file, <<EOF)
-/changed_managed_files/files/change=md5,size
-/changed_managed_files/files/name=/bar
+/changed_managed_files/change=md5,size
+/changed_managed_files/name=/bar
 EOF
 
         filter = subject.parse("inspect", "exclude" => "@#{exclude_file}")
 
         expect(filter.to_array).to match_array([
-          "/changed_managed_files/files/change=md5,size",
-          "/changed_managed_files/files/name=/bar"
+          "/changed_managed_files/change=md5,size",
```

Comment:
Use 2 spaces for indentation in an array, relative to the first position after the preceding left parenthesis.